udf/event_code_points_to_string/udf.sql

```diff
@@ -0,0 +1,43 @@
+CREATE OR REPLACE FUNCTION udf.event_code_points_to_string(code_points ANY TYPE) AS (
+  CODE_POINTS_TO_STRING(
+    IF(
+      code_points IS NULL,
+      NULL,
+      ARRAY(
+        SELECT
+          CASE
+          WHEN
+            i IS NULL
+          THEN
+            NULL
+            -- Skip commas
+          WHEN
+            i >= 43
+          THEN
+            i + 2
+            -- Skip double quote
```

Comment:
We use double quotes to represent "any" for an event property. It is one of the lowest-valued representable (not e.g. `\u0001`) unicode values that is not a meta-character in regex.